apps/cyberstorm-storybook/stories/components/Button.stories.tsx

```diff
@@ -0,0 +1,111 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Button } from "@thunderstore/cyberstorm";
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSkull, faChevronDown } from "@fortawesome/free-solid-svg-icons";
+
+const meta = {
+  title: "Cyberstorm/Components/Button",
+  component: Button,
+} as ComponentMeta<typeof Button>;
+
+const Template: ComponentStory<typeof Button> = (args) => <Button {...args} />;
+
+const DefaultButton = Template.bind({});
+DefaultButton.args = {
```

Comment:
Could save a lot of repetition with something like:

```typescript
const defaultArgs = {
  label: "Categories",
  rightIcon: (
    <FontAwesomeIcon
      fixedWidth={true}
      icon={faChevronDown}
      className={"buttonIcon"}
    />
  ),
};

...
DefaultButton.args = {...defaultArgs, buttonStyle: "default"};
```

Reply:
Very good, refactored to this.